src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -343,7 +345,8 @@ object LeonardoJsonSupport extends SprayJsonSupport with DefaultJsonProtocol {
         "userJupyterExtensionConfig" -> obj.userJupyterExtensionConfig.toJson,
         "dateAccessed" -> obj.auditInfo.dateAccessed.toJson,
         "autopauseThreshold" -> obj.autopauseThreshold.toJson,
-        "defaultClientId" -> obj.defaultClientId.toJson
+        "defaultClientId" -> obj.defaultClientId.toJson,
+        "stopAfterCreation" -> Option(obj.stopAfterCreation).toJson
```

Comment:
I think I made a mistake above and it should've been

```scala
"stopAfterCreation" -> Option(obj.stopAfterCreation).toJson
```

I will manually test the API JSON serialization/deserialization, too.